plugins/podspread/podspread.go

```diff
@@ -213,17 +214,29 @@ func getSpreadMode(nodeList *corev1.NodeList) (mode SpreadMode, regions, zones S
 	return
 }
 
-func calcRedunduncy(...any) int {
-	return 4
+func calcRedunduncy(totalReplicas int, rate float64) int {
+	return int(float64(totalReplicas) * rate)
 }
 
```

Comment:
```suggestion
const annotationPodSpreadRate = "podspread/rate"
```
※あとで設定ファイルから変更できるようにしたいです